### PR TITLE
Timeline styling

### PR DESCRIPTION
**Pull request by Markvanderuit**
Styling and layout for the timeline view. Some notes:
- Colors will come later. Black/white is fine for now, layout is more important.
- I'll work on the drawer sidepanel later, as that influences the entire project.
- Feel free to comment on the horrible code quality, this is my first time doing JS/css/etc in years, and you probably still owe me coffee anyways.

### CODE REVIEW

**Comment by bart-de-jonge** [comment]
Looks fine to me

File: public/js/timeline.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"use strict";
```
Comment (geenen124) [inline public/js/timeline.js]: This file should have been gitignored? (It's transpiled code)

Comment (Markvanderuit) [inline public/js/timeline.js]: Pretty sure that was already there. I'll add it to the ignore list.

Comment (bart-de-jonge) [inline public/js/timeline.js]: @Ezlo123 when a file is tracked just adding it to the ignore list doesn't help, that just prevents new files following those rules to be tracked. Use "git rm --cached [filename]" to stop tracking the file (it must be specified in the gitignore too, but it is i guess)

Comment (Markvanderuit) [inline public/js/timeline.js]: Interestingly enough, public/js/ is already in the gitignore.
I can add the file directly, ofc, but shouldn't it just be ignored in the first place?

**Comment by geenen124** [comment]
Looks great!

**Pull request merged**